Chain.js

```
@@ -0,0 +1,16 @@
+etherscan: {
+  enable: true,
+    apiKey: {
+      buildbear: "verifyContract",
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)